.github/workflows/go.yml

```diff
@@ -87,8 +71,12 @@ jobs:
     - name: Check-out code
       uses: actions/checkout@v2
 
-    - name: Build Antrea binaries
-      run: GOARCH=${{ matrix.arch }} make bin
+    - name: Build Antrea and antctl binaries
+      run: |
+        GOARCH=amd64 make bin
+        GOARCH=arm64 make bin
+        GOARCH=arm make bin
+        make antctl
```

Comment:
let's make those separate steps at least

Reply:
done

Reply:
@luolanzone I think we had a misunderstanding. I think you can still merge it in the same job if you want, but they should be different steps:
```yaml
steps:
  - name: Build Antrea binaries for amd64
    run: GOARCH=amd64 make bin
  - name: Build Antrea binaries for arm64
    run: GOARCH=arm64 make bin
  - name: Build Antrea binaries for arm
    run: GOARCH=arm make bin
  - name: Build antctl binaries
    run: make antctl
```
Up-to-you to decide. If you believe this will help, please make the change.

Reply:
I would prefer this way, just refined the yml, thanks